### ISSUE TITLE
VS Code 1.43 breaks Zowe Explorer's tree expand actions

### ISSUE THREAD

**Issue by phaumer** [zowe/vscode-extension-for-zowe#608]
After accepting the 1.43 update I need to expand/collapse all tree nodes twice before the children appear.

![2020-03-10_10-55-28](https://user-images.githubusercontent.com/4391934/76343881-f5c6aa00-62bd-11ea-85c0-ad9762cdbebe.gif)

**Comment by phaumer** [comment]
@Colin-Stone you now the tree event handling code much better, but seems that VS Code now behaves like Theia (that is a first).  So if we initialize the tree nodes with `dirty = true` then it would be fixed.

For example in `ZoweDatasetNode`

```ts
public dirty = true; // extension.ISTHEIA;  // Make sure this is true for theia instances
```

**Comment by phaumer** [comment]
Added PR to fix v1.3
